cmd/av/stack_sync.go

```diff
@@ -149,7 +149,6 @@ base branch.
 			return errors.New("refusing to sync: there are unstaged changes in the working tree (use `git add` to stage changes)")
 		}
 
-		var currentBranch string
```

Comment:
also found a bug where we weren't always setting `currentBranch` so to make the code a little simpler I just removed `currentBranch` and make sure we always set `state.CurrentBranch` and only refer to that.